doc/docker.md

```diff
@@ -1,8 +1,11 @@
 # Docker installation
 
+Before you begin, ensure that Docker is installed on your system. Follow the official Docker installation guide if it is not already installed: [Install Docker Engine](https://docs.docker.com/engine/install/).
```

Comment:
Maybe add a link to README of the dev container project? This way we won't repeat the documentation